src/v2/Apps/Settings/Routes/EditProfile/UserInformation.tsx

```diff
@@ -74,11 +73,18 @@ export const UserInformation: React.FC<UserInformationProps> = ({
 
   return (
     <Box>
-      <Serif size="6" mb={space(2)}>
+      <Text size="6" mb={2}>
```

Comment:
Get rid of the `size` prop and give this `variant="lg"`

Reply:
Done.